### ISSUE TITLE
Q2 Wording

### ISSUE THREAD

**Issue by Lord-of-Bugs** [darkprinx/break-the-ice-with-python#106]
It says to the program should print a comma-separated sequence yet it also says (and as we all know with common sense) that output should just a single integer. So should we correct it to the "program should print the single integer result" or something like this?

**Comment by jbakchr** [comment]
I totally agree with this. The wording does unfortunately make no sense and is in itself contradictory.

**Comment by darkprinx** [comment]
Checked the current Q2 wording — it no longer mentions a comma-separated sequence, just says the output should be a single integer (matching your suggestion). Looks like this was already cleaned up since you filed this. Closing as resolved — thanks for flagging it!
